hoomd/filter/set_.py

```diff
@@ -0,0 +1,35 @@
+from hoomd.filter.filter_ import ParticleFilter
+from hoomd import _hoomd
+
+
+class ParticleFilterSetOperations(ParticleFilter):
+    def __init__(self, f, g):
+        if f == g:
+            raise ValueError("Cannot use same filter for {}"
+                             "".format(self.__class__.__name__))
+        else:
+            self._f = f
+            self._g = g
+        getattr(_hoomd, self._cpp_cls_name).__init__(self, f, g)
```

Comment:
I see what this is supposed to do, but I would like a comment above this line explaining it.

Reply:
Can do.